otter/integration/tests/test_convergence.py

```diff
@@ -67,6 +70,72 @@ def _check_fds(_):
             return deferLater(reactor, 0, _check_fds, None)
         return self.pool.closeCachedConnections().addBoth(_check_fds)
 
+    def test_scale_over_group_max_after_metadata_removal(self):
+        """
+        Attempt to scale over the group max, but only after metadata removal on
+        some random sampling of servers.
+        """
+
+        rcs = TestResources()
+
+        scaling_group_body = create_scaling_group_dict(
+            image_ref=image_ref, flavor_ref=flavor_ref,
+        )
+
+        self.scaling_group = ScalingGroup(
+            group_config=scaling_group_body,
+            pool=self.pool
+        )
+
+        self.scale_up_to_max = ScalingPolicy(
+            scale_by=25,
+            scaling_group=self.scaling_group
+        )
+
+        self.scale_beyond_max = ScalingPolicy(
+            scale_by=5,
+            scaling_group=self.scaling_group
+        )
+
+        def expect_403(failure):
+            failure.trap(APIError)
+            if failure.value.code != 403:
+                failure.raiseException()
+            return rcs
+
+        return (
+            self.identity.authenticate_user(
+                rcs,
+                resources={
+                    "otter": ("autoscale", "http://localhost:9000/v1.0/{0}"),
+                    "nova": ("cloudServersOpenStack",),
+                },
+                region=region,
+            ).addCallback(self.scaling_group.start, self)
+            .addCallback(self.scale_up_to_max.start, self)
+            .addCallback(self.scale_beyond_max.start, self)
+            .addCallback(self.scale_up_to_max.execute)
+            .addCallback(
+                self.scaling_group.wait_for_N_servers, 25, timeout=1800
+            ).addCallback(self.scaling_group.get_scaling_group_state)
+            .addCallback(self._choose_random_servers, 3)
+            .addCallback(self._remove_metadata, rcs)
+            .addCallback(lambda _: rcs)
+            .addCallback(self.scale_beyond_max.execute)
+            .addErrback(expect_403)
```

Comment:
There should probably be a check in here that we expect a 403, and not a success.  If `self.scale_beyond_max.execute` succeeds, the 403 check is skipped and the test might pass, and we probably want the test to fail, because the scaling policy should not execute?